packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1068,7 +1070,7 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
   }
 
   private _notifyColumnsResized(): void {
-    this.state.adjustedColumns.forEach(column => {
+    this.state.adjustedColumns.forEach((column) => {
```

Comment:
This shouldn't be changing--looks like an issue with mismatched prettier versions (maybe if you're using the same clone/worktree for v7 and v8?). Try running `yarn` then saving the file again.

Reply:
weird, yeah, will do